lib/pups/config.rb

```diff
@@ -113,7 +117,7 @@ def run
     end
 
     def run_commands
-      @config['run'].each do |item|
+      @config['run']&.each do |item|
```

Comment:
I added this because pups would throw an exception if executed without any `run` config, which is a valid though unlikely use case.